src/modules/playRoom/components/ChatBox.js

```diff
@@ -0,0 +1,35 @@
+import imgBtnSend from '../../../assests/images/send.png';
```

Comment:
I don't think we should use images as input. It increases the load time for the website and make the UI hard to synchronize with other parts. Instead, try using Antd Icons